### PR TITLE
testgrid: Update OpenShift dashboards

### PR DESCRIPTION
**Pull request by smarterclayton**
Inculdes the new broken dashboard type.

/hold

for review of the generator in https://github.com/openshift/ci-tools/pull/639

### CODE REVIEW

File: config/testgrids/openshift/redhat-openshift-ocp-release-4.2-informing.yaml [hunk 1]
```diff
@@ -19,14 +19,14 @@ dashboards:
       - key: comment
         value: 'test: <test-name> failed, see job: <link>'
       url: https://bugzilla.redhat.com/enter_bug.cgi
-    name: release-openshift-ocp-installer-console-aws-4.2
+    name: promote-release-openshift-machine-os-content-e2e-aws-4.1
```
Comment (bparees) [inline config/testgrids/openshift/redhat-openshift-ocp-release-4.2-informing.yaml:22]: why are the promote jobs considered release informing?

**Comment by bparees** [comment]
/lgtm
(but i don't think i have merge rights)

**Comment by smarterclayton** [comment]
/hold cancel

Verified this matches latest, we are gtg

**Pull request merged**